src/index.js

```diff
@@ -99,7 +100,19 @@ const updateDailySummaryTable = async (sprint, pointsLeft) => {
   });
 };
 
-const getPointsLeftByDay = async (sprint, start) => {
+const getChartDatasets = async (sprint, start, includedEnd) => {
+  // set the end one day later to include the end
+  // note that the new variable end is the excluded day
+  // if the sprint is 1 to 4 June (inclusive), then
+  // start = 1 June, 0000h
+  // includedEnd = 4 June, 0000h
+  // end = 5 June, 0000h --> this includes the entire day of 4 June
+  /** date after the actual end date (excluded) */
+  const end = moment(includedEnd).add(1, "days");
+  const numDaysInSprint = moment(end).diff(start, "days");
+  // cool way to generate numbers from 1 to n
+  const labels = [...Array(numDaysInSprint).keys()].map((i) => i + 1);
```

Comment:
Is there a reason why you wanted to generate the labels in this function? Looks like it's not being used in this function at all

Reply:
not for the labels, but for `numDaysInSprint` cause that is used for generating both labels and the guideline